File: py/7.py
from math import sqrt


def is_prime(num):
    for i in range(2, int(sqrt(num) + 1)):
        if num % i == 0:
            return False

    return True


counter = 1
i = 3

while counter < 10001:
    if is_prime(i):
        counter += 1

    i += 2

print(i - 2)

File: cpp/7.cpp
#include <bits/stdc++.h>
using namespace std;

bool isPrime(int num) {
  for (int i = 2; i <= sqrt(num); i++)
    if (num % i == 0) return false;

  return true;
}

int main() {
  int counter = 1, i = 3;

  for (; counter < 10001; i += 2)
    if (isPrime(i)) ++counter;

  printf("%d\n", i - 2);
}
